### ISSUE TITLE
Combobox/DateField/Spinner keypress events stop firing after all characters selected by 'swiping' across them

### ISSUE THREAD

**Issue by mrfeet** [qooxdoo/qooxdoo#9331]
I have found that the key pressed listener is not always triggered when the text in the field is selected by clicking and swiping across the text.

This has been reported before for textfield and textarea (#2737), but compound widgets containing a textfield have not been taken into account.

So I have extended the example from issue #2737 to show the problem for ComboBox, DateField and Spinner:

```javascript
var inputWidget = new qx.ui.form.ComboBox();    // Fails
//var inputWidget = new qx.ui.form.DateField();   // Fails
//var inputWidget = new qx.ui.form.Spinner();     // Fails
//var inputWidget = new qx.ui.form.TextField();   // Goes ok
//var inputWidget = new qx.ui.form.TextArea();    // Goes ok
inputWidget.addListener("keypress", function(e)
  {
      alert("keypress");
  }, this);
  
// Document is the application root
var doc = this.getRoot();

// Add button to document at fixed coordinates
doc.add(inputWidget,
{
  left : 100,
  top  : 50
});
```

**Comment by level420** [comment]
Should be fixed via PR #9333 and followup PR #9347. We will have to wait until PR #9347 is merged.

**Comment by level420** [comment]
PR #9347 is merged
closing
